### PR TITLE
update(OWNERS): add loresuso to approvers

### PR DESCRIPTION
**Pull request by loresuso**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

> /kind flaky-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area library

> /area cli

> /area tests

> /area examples

**What this PR does / why we need it**:

I am opening this PR after proposing myself as falcoctl maintainer in today's community call, starting the voting process!
cc @falcosecurity/falcoctl-maintainers 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -5,8 +5,8 @@ approvers:
   - fededp
   - cpanato
   - alacuku
-reviewers:
   - loresuso
+
```
Comment (leogr) [inline OWNERS:9]:
```suggestion
```

**Comment by leogr** [comment]
ofc big +1 from me! :star_struck: 

Lore, thank you! :hugs:

**Comment by FedeDP** [comment]
You surely deserve it! +1 on my side!

**Comment by alacuku** [comment]
+1 on my side!

**Comment by cpanato** [comment]
+1

**Review by leogr — APPROVED** [review]
One week has passed and we also have the majority (4/6).

:partying_face: 

/hold cancel

**Pull request merged**